v3/lints/cabf_ev/lint_ev_certificate_policies.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * ZLint Copyright 2024 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package cabf_ev
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/zmap/zcrypto/x509"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+func init() {
+	lint.RegisterCertificateLint(&lint.CertificateLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_ev_certificate_policies",
+			Description:   "EV Certificates issued to Subscribers MUST include a CPS URI policy qualifier",
+			Citation:      "CA/Browser Forum EV Guidelines v1.8.1, Sec. 9.7.3",
```

Comment:
I think this has now changed to 7.1.4.3: https://github.com/cabforum/servercert/blob/main/docs/EVG.md#7143-additional-technical-requirements-for-ev-certificates